src/cascadia/TerminalApp/TerminalPage.cpp

```diff
@@ -1307,11 +1334,11 @@ namespace winrt::TerminalApp::implementation
             auto newControl = _InitControl(controlSettings, controlConnection);
 
             // Hookup our event handlers to the new terminal
-            _RegisterTerminalEvents(newControl, *focusedTab);
+            _RegisterTerminalEvents(newControl, *&tab);
```

Comment:
```suggestion
            _RegisterTerminalEvents(newControl, tab);
```
Won't it work without the `*&`? `_RegisterTerminalEvents()` already takes a `TerminalTab&`, and `tab` is just that.

Reply:
yea this looks insane as-is haha